src/de/uni_hamburg/informatik/swt/se2/mediathek/ui/ausleihe/AusleihWerkzeug.java

```diff
@@ -216,7 +227,12 @@ private boolean istAusleihenMoeglich()
         // Medien nur vom ersten Vormerker ausgeliehen werden können, gemäß
         // Anforderung c).
         boolean ausleiheMoeglich = (kunde != null) && !medien.isEmpty()
-                && _verleihService.sindAlleNichtVerliehen(medien);
+                && _verleihService.sindAlleNichtVerliehen(medien)
```

Comment:
Sieht gut aus, ggf. kurze Implementationskommentare + vllt auch diese Funktionalität (also ob das ausleihen möglich ist) nach VormerkService verlagern (ist ja nicht im Aufgabenbereich von AusleihWerkzeug)

Reply:
dann müssen wir das auch noch testen und dokumentieren; ich wollte das eigendlich nicht in dieser pr machen; villeicht kann das noch jemand refactorn, wenn die methode hinzugefügt wird.